packages/dev/core/src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -202,6 +210,17 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
         this._hitPlane = Plane.FromPositionAndNormal(camera.target, direction);
     }
 
+    private _updateCameraTarget() {
+        const camera = this.camera;
+        const direction = camera.target.subtract(camera.position).normalize();
+        const ray = new Ray(camera.position, direction, Number.MAX_SAFE_INTEGER);
+        const ground = Plane.FromPositionAndNormal(Vector3.Zero(), camera.upVector);
+        const distance = ray.intersectsPlane(ground) ?? 0;
+        const intersectionPoint = ray.origin.add(ray.direction.scale(distance));
+        camera.setTarget(intersectionPoint);
+        camera.targetSetManually = false;

```

Comment:
It looks like this value is set outside of the code (possibly by the user) to allow the user to force the target to be set for a single frame and then is reset to `false`.  What is the use case for this to be set to true?  Is it something to be set to `true` once (as needed) or repeated more frequently?

Reply:
This value can now be set to true when passed in by setTarget.

The use case that is true is :
to tell the mouse wheel event that the user forcibly changed the camera's target, 
and that the camera's reasonable target needs to be updated once to ensure proper use of zoomToMouseLocation